src/components/AddTaskField/AddTaskField.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import ShowError from '../ShowError/ShowError';
+import './style.scss';
+import { addTask } from '../../service/Requests';
+
+const AddTaskField = ({ tasks, setTasks, error, setError }) => {
```

Comment:
logically, this component should not know about all tasks

and the "tasks" state must be changed where it is created, otherwise it will be difficult to track its changes throughout the project